src/gui/DatabaseSettingsWidget.cpp

```diff
@@ -23,8 +23,13 @@
 #include "core/Metadata.h"
 #include "crypto/SymmetricCipher.h"
 #include "format/KeePass2.h"
+#include "gui/MessageBox.h"
 #include "keys/CompositeKey.h"
 
+namespace {
+constexpr int Megabyte = 1024 * 1024;
```

Comment:
Technically, it's a Mebibyte. 😉

Honestly, I wouldn't mind if you just hard-coded 1024 * 1024 in those two places where you need it.